### PR TITLE
feat(DTFS2-6023): Facility amendment SOFs

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Facility amendment could consists of both value increment or value decrement.
Value increment works as expected, however decrement require facility records amendment to be swapped.

In a standard value increment records are amended in following sequence:

* FMR
* FLR

However, in case of decrement records are amended in reverse:

* FLR
* FMR

## Resolution
* Execution sequence are swapped upon operation type.
* GEF facility amendment bug resolution with regards to 10% of the amendment value and condition update for payload construction.

## Miscellaneous
* Documentation update

### CODE REVIEW

File: azure-functions/acbs-function/acbs-amend-facility-loan-record/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ *
+ * ACBS
+ * ----
+ * This function is responsible for amending FLR.
+ * FLR is one of a child record of an issued facility.
+ */
+const df = require('durable-functions');
+const retryOptions = require('../helpers/retryOptions');
+const mappings = require('../mappings');
+const CONSTANTS = require('../constants');
```
Comment (IgnatG) [inline azure-functions/acbs-function/acbs-amend-facility-loan-record/index.js:26]: If we only need the Facility, then we can destructure it here

File: azure-functions/acbs-function/acbs-amend-facility-master-record/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ * This is a sub-orchestrated function invoked from it's master orchestrator.
+ * `acbs-amend-facility` is the master orchestrator for this function.
+ *
+ * Sub-Orchestration Function (SOF)
+ * --------------------------------
+ * This function cannot be invoked directly and must be invoked by a DOF.
+ *
+ * Prerequisites
+ * -------------
+ * 0. 'npm install durable-functions'
```
Comment (IgnatG) [inline azure-functions/acbs-function/acbs-amend-facility-master-record/index.js:13]: Do we need this line?

Comment (abhi-markan) [reply azure-functions/acbs-function/acbs-amend-facility-master-record/index.js:13]: Yep, especially for a clear documentation.

File: azure-functions/acbs-function/acbs-amend-facility-loan-record/index.js (hunk 1 above)
Comment (abhi-markan) [reply azure-functions/acbs-function/acbs-amend-facility-loan-record/index.js:26]: Good spot, have updated.

**Review by IgnatG — APPROVED** [review]
LGTM

**Pull request merged**